### ISSUE TITLE
`constants.FLOAT128` type fails on Windows 

### ISSUE THREAD

**Issue by hobson** [totalgood/nlpia#18]
In contants.py need to define types in a cross-platform way:

```python
>>> FLOAT_TYPES = [t for t in set(np.typeDict.values()) if t.__name__.startswith('float')]
>>> FLOAT_TYPES
[numpy.float64, numpy.float128, numpy.float16, numpy.float32]
>>> FLOAT_TYPE_NAMES = [t.__name__ for t in FLOAT_TYPES]
>>> FLOAT_TYPE_NAMES
['float64', 'float128', 'float16', 'float32']
```

**Comment by hobson** [comment]
Fixed with pugnlp release 0.1.11

**Comment by hobson** [comment]
Fixed with pugnlp release 0.1.11
